compiler/src/dmd/cpreprocess.d

```diff
@@ -104,6 +91,34 @@ FileName preprocess(FileName csrcfile, ref const Loc loc, out bool ifile, OutBuf
         return csrcfile;        // no-op
 }
 
+
+/***************************************
+ * Find importc.h by looking along the path
+ * Params:
+ *      path = import path
+ * Returns:
+ *      importc.h file name, null if not found
+ */
+const(char)* findImportcH(const(char)*[] path)
```

Comment:
private, but otherwise looks good

Reply:
I'm planning on calling it from dmodule.d, so left it public.

Reply:
This might not be the best place to put it then as this isn't a common frontend module.